docs/INSTALL.md

```diff
@@ -345,4 +347,6 @@ Bitcoin chain. `lnd` also supports Litecoin testnet4 (but not both BTC and LTC
 at the same time), so when working with Litecoin be sure to set to parameters
 for Litecoin accordingly. For node configuration, the sections are called
 `[Btcd]`, `[Bitcoind]`, `[Neutrino]`, `[Ltcd]`, and `[Litecoind]` depending on
-which chain and node type you're using.
+which chain and node type you're using.  
+Make sure to place options in the proper section.  
```

Comment:
might be worth elaborating that "proper section" means that it must be under the section with the option's prefix, e.g. `bitcoind.xxxx` should go in `[Bitcoind]`

Reply:
Hmm that is not always the case, other options don't have the prefix so it might get confusing?